### ISSUE TITLE
Unexpected internal error near index 1

### ISSUE THREAD

**Issue by jonneymendoza** [jvoegele/gradle-android-plugin#87]
Hi. i copied the example build.gradle script and tried to run it but seem to be getting this error:

sdkDir = C:/Program Files (x86)/Android/android-sdk
toolsDir = C:\Program Files (x86)\Android\android-sdk\tools
platformToolsDir = C:\Program Files (x86)\Android\android-sdk\platform-tools
:androidProcessResources FAILED

FAILURE: Build failed with an exception.
- What went wrong:
  Execution failed for task ':androidProcessResources'.
  
  > Unexpected internal error near index 1
  > \
  > ^
- Try:
  Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 3.737 secs

My build.gradle looks like this:

//setup external dependancy plugins we will use to build a android application
buildscript {
    repositories {
        mavenCentral()
        maven {
            url "https://oss.sonatype.org/content/repositories/snapshots"
          }
    }

```
dependencies {
classpath 'org.gradle.api.plugins:gradle-android-plugin:1.2.1'
}
```

}

//apply eclipse plugin
apply plugin: 'eclipse'

//apply android plugin
apply plugin: 'android'

apply plugin: 'maven'

//set src folder
sourceSets {
    main {
      java {
        srcDir 'src'
      }
    }
  }

repositories {
    mavenCentral()
}

// Sets the package version
version = "1.0.0"

//test task
task hello <<{
    String value = 'wagwan'

```
    println 'Hello world!' + value.toUpperCase()
```

}

//signing the app
androidSignAndAlign{
    keyStore = "path/to/my/keystore"
    keyAlias = "my-key-alias"
    keyStorePassword = "mystorepass"
    keyAliasPassword = "myaliaspass"
}

// Configure the filtering of resources with properties from the Gradle's project scope (2)
processResources {
  expand (project.properties)
}

// Configure a dedicated debug build (3)
task configureDebug << {
  jar.classifier = "debug"
}

// Configure a dedicated release build (4)
task configureRelease << {
  proguard.enabled = true
}

Thanks

**Comment by jonneymendoza** [comment]
Anyone have a solution to this?

**Comment by mrosseel** [comment]
knowing nothing about either gradle or gradle for android, it seems to me that the slash notation of your paths might be the cause of this error. Try replacing your \'s by /

**Comment by sonata82** [comment]
I think this has been fixed with [commit db47e69a3dd1ba3faba480d6424e5e233409de26](https://github.com/jvoegele/gradle-android-plugin/commit/db47e69a3dd1ba3faba480d6424e5e233409de26#src/main/groovy/com/jvoegele/gradle/tasks/android/ProcessAndroidResources.groovy)

Unfortunately version 1.2.1 from Maven central does not have this yet :-(
